develop/components/pages/TablePage.tsx

```diff
@@ -0,0 +1,228 @@
+            },
+            {
+                label: "Last Name",
+                accessor: "lastName",
+            },
+            {
+                label: "Age",
+                accessor: "age",
+            },
+            {
+                label: "Visits",
+                accessor: "visits",
+            },
+            {
+                label: "Profile Progress",
+                accessor: "progress",
+            },
+            {
+                label: "Status",
+                accessor: "status",
+            },
+        ],
+        []
+    );
+    const data: Array<DataItem> = React.useMemo(() => makeData<Array<DataItem>>([listSize, 5]), []);
+    const smallData: Array<DataItem> = React.useMemo(() => makeData<Array<DataItem>>([5, 5]), []);
+    return (
+        <div className="route-template container">
+            <div className="info-holder">
+
+                <div className="info">
+                    <div className="md-file">
+                        <Highlight innerHTML={true}>{docMD}</Highlight>
+                    </div>
+                </div>
+
+                <div className="info">
+                    <h2>Output</h2>
+
+                    <p>Here are sample outputs of plain table</p>
+                    <div className="result wide">
+                        <Table
+                            columns={columns}
+                            data={smallData}
+                        />
+                    </div>
+
+                    <p>Here an example with sorting</p>
+                    <div className="result wide">
+                        <Table
+                            columns={columns}
+                            data={smallData}
+                            sortProps={{
+                                onAfterSorting: (rows: Array<TableRow>, sortByColumn: TableHeader) => { console.log("The sorted rows are ", rows); }
+                            }
+                            }
+                        />
+                    </div>
+
+                    <p>Here an example with pagination</p>
+                    <div className="result wide">
+                        <Table
+                            columns={columns}
+                            data={data}
+                            offset={pageSize}
+                            currentpage={paginationValue}
+                            footer={
+                                <Pagination
+                                    value={paginationValue}
+                                    onChange={setPagination}
+                                    size={listSize}
+                                    useFirstAndLast={true}
+                                />
+                            }
+                        />
+                    </div>
+
+                    <p>Here is an example with expandable subrows and rowDetails</p>
+                    <div className="result wide">
+                        <Table
```

Comment:
This table have a bug, those are the steps to reproduce:
1. expand the first item in the row.
2. expand any item if the subrow
3. shrink the row.
the details "Lorem ipsum" is still displayed.

Im not sure if this is the expected behavior.